Makefile

```diff
@@ -79,8 +79,7 @@ debug:
 GO_BUILD_FLAGS :=-tags 'include_gcs include_oss containers_image_openpgp gssapi providerless netgo osusergo'
 
 # targets "all:" and "build:" defined in vendor/github.com/openshift/build-machinery-go/make/targets/golang/build.mk
-microshift: build-containerized-cross-build-linux-amd64
-.PHONY: microshift
+microshift: build
```

Comment:
microshift was a phony target (no binary output associated) but we really have a ./microshift output which is handled by `make`,  now make microshift can handle that too, and other actions can depend on that if the binary doesn't exist in ./